packages/plugin/README.md

```diff
@@ -0,0 +1,7 @@
+# Theia - Plugin Extension
```

Comment:
FYI it may redirect to https://github.com/theia-ide/theia/blob/2d8bd9987b6b1eb257d1f08b8136f3344206fbe0/packages/plugin/API.md

Reply:
thank you @benoitf, it's been corrected